src/main/java/buildcraftAdditions/items/ItemBucketBCA.java

```diff
@@ -34,7 +37,8 @@ public ItemBucketBCA(Fluid fluid) {
 		setContainerItem(Items.bucket);
 		setCreativeTab(BuildcraftAdditions.bcadditions);
 		setUnlocalizedName("bcaBucket." + fluid.getName());
-		MinecraftForgeClient.registerItemRenderer(this, BucketItemRenderer.INSTANCE);
+		if (FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT)
```

Comment:
It will be very hacky to do this in the ClientProry. It will look very crowded and unelegant.